tests/azure/nic_test.go

```diff
@@ -28,7 +28,7 @@ func testNicCreate(t *testing.T, f fixture.F) {
 		f.ResGroupName,
 		f.Location,
 	)
-	_ = azure.NewNsg(f)
+	azure.NewNsg(f)
```

Comment:
I was hinting that you dont need to call any New* methods if you are not going to use the return value. New functions are usually constructors, they dont have side effects (or should not at least), just calling it and ignoring the return is odd, and in this case I checked out the New functions and they do almost nothing. No need to call them here ;-)